fms/utils/generation.py

```diff
@@ -4,6 +4,88 @@
 import torch.nn.functional as F
 
 
+def __create_prefill_mask(
+    prompt_lengths: List[int], device: Union[str, torch.device]
+) -> torch.Tensor:
+    """Create a prefill mask based on prompt_lengths
```

Comment:
I don't think our definitions of 'prefill' and 'decode' will be obvious to all readers, and may be confusing since some will assume 'decode' refers to 'encoder/decoder'

Reply:
agreed, you could just say: a first decoding iteration mask (also known as prefill step)

Reply:
added more information in documentation